spec/state_of_the_nation_spec.rb

```diff
@@ -210,48 +231,97 @@ class Country < ActiveRecord::Base
-    let!(:cochran) { country.senators.create!(entered_office_at: day(1978)) }
+    context "with senators" do
+      let!(:byrd) { country.senators.create!(entered_office_at: day(1959), left_office_at: day(2010)) }
+      let!(:inouye) { country.senators.create!(entered_office_at: day(1963), left_office_at: day(2012)) }
+      let!(:thurmond) { country.senators.create!(entered_office_at: day(1954), left_office_at: day(2003)) }
+      let!(:kennedy) { country.senators.create!(entered_office_at: day(1962), left_office_at: day(2009)) }
+      let!(:cochran) { country.senators.create!(entered_office_at: day(1978)) }
+
+      it "defaults to now" do
+        travel_to(day(1960)) do
+          expect(Senator.active).to eq [byrd, thurmond]
+
+          expect(byrd).to be_active
+          expect(kennedy).not_to be_active
+        end
+      end
 
-    it "defaults to now" do
-      travel_to(day(1960)) do
-        expect(Senator.active).to eq [byrd, thurmond]
+      it "works" do
+        expect(Senator.active(day(2008))).to eq [byrd, inouye, kennedy, cochran]
 
-        expect(byrd).to be_active
-        expect(kennedy).not_to be_active
+        expect(country.active_senators(day(2008))).to eq [byrd, inouye, kennedy, cochran]
+
+        expect(byrd).to be_active(day(2008))
+        expect(thurmond).not_to be_active(day(2008))
       end
-    end
 
-    it "works" do
-      expect(Senator.active(day(2008))).to eq [byrd, inouye, kennedy, cochran]
+      it "works on the boundaries" do
+        expect(Senator.active(day(1954))).to include thurmond
+        expect(Senator.active(day(2003))).not_to include thurmond
 
-      expect(country.active_senators(day(2008))).to eq [byrd, inouye, kennedy, cochran]
+        expect(country.active_senators(day(1954))).to include thurmond
+        expect(country.active_senators(day(2003))).not_to include thurmond
 
-      expect(byrd).to be_active(day(2008))
-      expect(thurmond).not_to be_active(day(2008))
-    end
+        expect(thurmond).to be_active(day(1954))
+        expect(thurmond).not_to be_active(day(2003))
+      end
 
-    it "works on the boundaries" do
-      expect(Senator.active(day(1954))).to include thurmond
-      expect(Senator.active(day(2003))).not_to include thurmond
+      it "works when nothing active" do
+        expect(Senator.active(day(1600))).to eq []
 
-      expect(country.active_senators(day(1954))).to include thurmond
-      expect(country.active_senators(day(2003))).not_to include thurmond
+        expect(country.active_senators(day(1600))).to eq []
 
-      expect(thurmond).to be_active(day(1954))
-      expect(thurmond).not_to be_active(day(2003))
+        [byrd, inouye, thurmond, kennedy, cochran].each do |senator|
+          expect(senator).not_to be_active(day(1600))
+        end
+      end
     end
 
-    it "works when nothing active" do
-      expect(Senator.active(day(1600))).to eq []
+    context "with members of congress" do
```

Comment:
This is mostly a duplicate of the tests for senators above. I was tempted to use some shared examples but they quickly became unclear and complex. Going verbose and simple here.